src/shared-components/banner/style.ts

```diff
@@ -25,8 +27,21 @@ const errorAlertStyles = css`
   box-shadow: 0px 8px 24px rgba(189, 32, 15, 0.05);
 `;
 
-export const BannerContainer = styled.div<{ bannerType: BannerType }>`
-  cursor: pointer;
+/**
+ * width: 100% to match previous <div> behavior
```

Comment:
[super-nit, sorry!] I think this comment actually makes things more confusing, because you need to look at the history to see that this was previously a `div`. I think it will be clear that removing the `width` would cause a style regression.